python/ray/serve/_private/autoscaling_policy.py

```diff
@@ -1,16 +1,20 @@
 import logging
 import math
 from abc import ABCMeta, abstractmethod
-from typing import List
+from typing import List, Optional
 
+from ray.serve._private.common import TargetCapacityScaleDirection
 from ray.serve._private.constants import CONTROL_LOOP_PERIOD_S, SERVE_LOGGER_NAME
+from ray.serve._private.utils import get_capacity_adjusted_num_replicas
 from ray.serve.config import AutoscalingConfig
 
 logger = logging.getLogger(SERVE_LOGGER_NAME)
 
 
 def calculate_desired_num_replicas(
-    autoscaling_config: AutoscalingConfig, current_num_ongoing_requests: List[float]
+    autoscaling_config: AutoscalingConfig,
+    current_num_ongoing_requests: List[float],
+    override_min_replicas: Optional[float] = None,
```

Comment:
Should there also be an `override_max_replicas`?

Reply:
Thanks for the catch, I added it.